armi/mpiActions.py

```diff
@@ -314,28 +314,82 @@ def runActions(o, r, cs, actions, numPerNode=None, serial=False):
     batchNum = 0
     while queue:
         actionsThisRound = []
-        for useRank in useForComputation:
-            actionsThisRound.append(queue.pop(0) if useRank and queue else None)
-        useForComputation = _disableForExclusiveTasks(actionsThisRound, useForComputation)
-        realActions = [
-            (context.MPI_NODENAMES[rank], rank, act) for rank, act in enumerate(actionsThisRound) if act is not None
-        ]
         batchNum += 1
         runLog.extra(
-            "Distributing {} MPI actions for parallel processing (batch {} of {}):\n{}".format(
-                len(realActions),
+            "MPI actions, batch {} of {}:\n".format(
```

Comment:
Non-blocking, I prefer f-strings in ARMI.  (I know there are a lot of `.format()` strings in ARMI, there is a plan to remove them, it's just not priority.)